src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,85 @@
+package duke;
+import java.util.Scanner;
+
+public class Duke {
+
+    Scanner in = new Scanner(System.in);
+    private String commands;
+    Task[] tasks = new Task[100];
+
+    Duke() {
+        System.out.println("Hello, I am Duke!");
+        System.out.println("What can I do for you?");
+    }
+
+    void processCommands(){
+        int i = 0;
+        while(true) {
+            commands = in.nextLine();
+            String[] words = commands.split(" ");
+            if (commands.equals("list")) {
+                processList(i);
+            } else if (commands.equals("exit")) {
+                processExit();
+                break;
+            } else if (words[0].equals("todo")) {
+                processTodo(i);
+                i++;
+            } else if (words[0].equals("deadline")) {
+                processDeadline(i);
+                i++;
+            } else if (words[0].equals("event")) {
+                processEvent(i);
+                i++;
+            } else if(words[0].equals("done")){
+                processDone(i, words);
+            }
```

Comment:
Is there a particular reason why you decided to use if-elseif-else rather than switch here?